com.civicrm.hrjobroles/tests/phpunit/CRM/Hrjobroles/BAO/HrJobRolesTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Civi\Test\HeadlessInterface;
+use Civi\Test\TransactionalInterface;
+
+/**
+ * Class CRM_Hrjobroles_BAO_HrJobRolesTest
+ *
+ * @group headless
+ */
+class CRM_Hrjobroles_BAO_HrJobRolesTest extends \PHPUnit_Framework_TestCase implements HeadlessInterface, TransactionalInterface {
+
+  use HrJobRolesTestTrait;
+
+  public function setUpHeadless() {
+    return \Civi\Test::headless()
+      ->install('org.civicrm.hrjobcontract')
+      ->installMe(__DIR__)
+      ->apply();
+    $jobContractUpgrader = CRM_Hrjobcontract_Upgrader::instance();
+    $jobContractUpgrader->install();
+  }
+
+  public function testGetDepartmentsList() {
+    $contactParams = array("first_name" => "chrollo", "last_name" => "lucilfer");
+    $contactID =  $this->createContact($contactParams);
+    $contract =  $this->createJobContract($contactID);
+    $departmentID1 =  $this->createDepartment('special_investigation', 'Special Investigation');
+    $params = array('job_contract_id'=>$contract->id, 'department'=>$departmentID1);
```

Comment:
Missing space between array keys and values. Also, some space between the statements would certainly improve readability